mrp_bom_line_formula_quantity/models/mrp_production.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2024 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MRPProduction(models.Model):
+    _inherit = "mrp.production"
+
+    def _get_move_raw_values(
+        self,
+        product_id,
```

Comment:
Can you adjust to `product` as done upstream? https://github.com/odoo/odoo/commit/cf48e39629cd0bf726da87bcdcc4df88790e81a0

Reply:
@LoisRForgeFlow Done